sdk/storage/Azure.Storage.Files.DataLake/tests/DirectoryClientTests.cs

```diff
@@ -350,6 +350,109 @@ public async Task CreateAsync_ConditionsFail()
             }
         }
 
+        [RecordedTest]
+        [ServiceVersion(Min = DataLakeClientOptions.ServiceVersion.V2020_12_06)]
+        public async Task CreateAsync_EncryptionScopeSas()
+        {
+            // Arrange
+            await using DisposingFileSystem test = await GetNewFileSystem();
+            string directoryName = GetNewDirectoryName();
+
+            DataLakeSasBuilder sasBuilder = new DataLakeSasBuilder
+            {
+                FileSystemName = test.FileSystem.Name,
+                Path = directoryName,
+                EncryptionScope = TestConfigHierarchicalNamespace.EncryptionScope,
+                ExpiresOn = Recording.UtcNow.AddDays(1)
+            };
+            sasBuilder.SetPermissions(DataLakeSasPermissions.All);
+
+            DataLakeSasQueryParameters sasQueryParameters = sasBuilder.ToSasQueryParameters(
+                new StorageSharedKeyCredential(
+                    TestConfigHierarchicalNamespace.AccountName,
+                    TestConfigHierarchicalNamespace.AccountKey));
+
+            DataLakeUriBuilder uriBuilder = new DataLakeUriBuilder(test.FileSystem.Uri)
+            {
+                DirectoryOrFilePath = directoryName,
+                Sas = sasQueryParameters
+            };
+
+            DataLakeDirectoryClient sasDirectoryClient = InstrumentClient(new DataLakeDirectoryClient(uriBuilder.ToUri(), GetOptions()));
+
+            // Act
+            Response<PathInfo> response = await sasDirectoryClient.CreateAsync();
+
+            // Assert
+            // TODO
+        }
+
+        [Ignore("TODO need to figure this one out")]
```

Comment:
Huge TODO, I haven't gotten Account SAS Encryption Scopes to work with Data Lake yet.